styles/components/_hero.scss

```diff
@@ -0,0 +1,47 @@
+.hero {
+  background: $tundora 100% / cover;
```

Comment:
The background here doesn't actually matter, because we're no longer messing with opacity or blur on these images